magenta/scripts/convert_dir_to_note_sequences.py

```diff
@@ -75,7 +75,7 @@ def convert_directory(root_dir, sub_dir, sequence_writer, recursive=False):
   sequences_written = 0
   sequences_skipped = 0
   for file_in_dir in files_in_dir:
-    full_file_path = os.path.join(dir_to_convert, file_in_dir)
+    full_file_path = os.path.join(dir_to_convert, file_in_dir).lower()
```

Comment:
Converting to lower case should happen only for the file extension checks. If it happens here, the next line may fail on a case sensitive file system.